.github/workflows/agw-workflow.yml

```diff
@@ -146,20 +146,10 @@ jobs:
     name: C/C++ unit tests with Bazel
     runs-on: ubuntu-latest
     steps:
-      - name: Maximize build space
-        shell: bash
-        run: |
-          echo "Available storage before:"
-          df -h
-          echo "Removing /usr/share/dotnet, /usr/local/lib/android, /opt/ghc"
-          sudo rm -rf /usr/share/dotnet
-          sudo rm -rf /usr/local/lib/android
-          sudo rm -rf /opt/ghc
-          echo "Available storage after:"
-          df -h
       - name: Check Out Repo
         # This is necessary for overlays into the Docker container below.
         uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e # pin@v2
+      - uses: ./.github/workflows/composite/maximize-build-space
```

Comment:
Nitpick: All the other steps have a `name: ` entry with a descriptive name, we could adhere to that pattern.

Reply:
Done. (Currently almost all local action calls do not have a name field.)